ingredient.py

```diff
@@ -7,11 +19,14 @@ def __init__(self, amount, unit, name):
 		self.name = name
 
 	def convert_to_decimal(self, amount):
-	    try:
-	        return float(amount)
-	    except ValueError:
-	        num, denom = amount.split('/')
-	        return float(num) / float(denom)
+		try:
+			return float(amount)
+		except ValueError:
+			num, denom = amount.split('/')
+			return float(num) / float(denom)
 
-	def convert_to_grams(self):
-		pass
\ No newline at end of file
+	def convert_to_grams(self, amount, unit):
+		return float(amount) * dict.get(per_gr_coversion, unit, 1)
+
+	def convert_from_grams(self, amount, unit):
+		return float(amount) / dict.get(per_gr_coversion, unit, 1)
```

Comment:
later functionality will be for us to select the highest out of cups, tbsps, and tsps ourselves and convert it to that instead of requiring the function caller to specify the unit that they want returned